pages.de/common/nix.md

```diff
@@ -1,24 +1,34 @@
 # nix
 
-> Dienstprogramme für die Nix-Sprache und den Nix-Speicher.
+> Ein leistungsfähiger Paketmanager, der das Paketmanagement zuverlässig, reproduzierbar und deklarativ macht.
+> `nix` ist experimentell und muss gesondert aktiviert werden. Für die klassische, stabile Schnittstelle siehe `tldr nix classic`.
+> Einige Unterbefehle wie `build`, `develop`, `flake`, `registry`, `profile`, `search`, `repl`, `store`, `edit`, `why-depends` usw. haben ihre eigene Benutzerdokumentation.
 > Weitere Informationen: <https://nix.dev/manual/nix/stable/command-ref/new-cli/nix>.
 
-- Suche nach einem Paket über seinen Namen oder seine Beschreibung:
+- Aktiviere den `nix` Befehl:
 
-`nix search {{suchbegriff}}`
+`mkdir {{[-p|--parents]}} ~/.config/nix; echo 'experimental-features = nix-command flakes' > ~/.config/nix/nix.conf`
 
-- Starte eine Nix-Shell, die die angegebenen Pakete zur Verfügung stellt:
+- Suche nach einem Paket in nixpkgs via Packagenamen oder Beschreibung:
```

Comment:
Sounds better to me, but I'll let you be the judge. Either way, please change „Packagename“ to „Paketname“ or just „Name“.

```suggestion
- Suche ein Paket in nixpkgs nach Namen oder Beschreibung:
```